### PR TITLE
docs(other): update readme.md

### LINKED ISSUE #6089: 💥 [DevOps] update readme.md
## 💥 DevOps ticket

update readme.md

### PR DESCRIPTION
**Pull request by sushidave**
Add general information

## 🍰 Pullrequest 

update readme.md

### Issues

- fixes #6089

### Todo

- [X] None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,71 @@ At the same time, it should be possible in the future to link these networks wit
...
+* [User Guide](/wiki/en:User-Guide)
+* [Frequently Asked Questions](/wiki/en:FAQ)
+
+## Demo
+
+Try out our live demo network, see [here](#live-demo-and-developer-logins).
+
+## Help us
+
+If you're wondering how could you help, there are plenty of ways, e.g.:
```
Comment (Mogge) [inline README.md:56]:
```suggestion
If you're wondering how you could help, there are plenty of ways, e.g.:
```

**Review by Mogge — CHANGES_REQUESTED** [review]
Thank you very much for improving our read me!!!

File: README.md [hunk 2]
```diff
@@ -20,12 +20,71 @@ At the same time, it should be possible in the future to link these networks wit
...
+
+Ocelot.social networks feature: 
+* <strong>user accounts</strong>
+* <strong>user roles</strong>
+* <strong>posts</strong> and <strong>comments</strong>
+* <strong>groups</strong>
+* <strong>maps</strong>
+* <strong>search</strong>
+* <strong>filters</strong>
+* ...and more.
```
Comment (Tirokk) [inline README.md:33]:
```suggestion
* and more …
```

**Review by Tirokk — CHANGES_REQUESTED** [review]

**Review by Tirokk — APPROVED** [review]
Hey @sushidave ,

very cool! 🚀💫

**Pull request merged**